apps/desktop/src/app/layout/account-switcher.component.html

```diff
@@ -55,38 +56,40 @@
     aria-modal="true"
   >
     <div class="accounts" *ngIf="numberOfAccounts > 0">
-      <button *ngFor="let a of accounts | keyvalue" class="account" (click)="switch(a.key)">
+      <button
+        *ngFor="let account of inactiveAccounts | keyvalue"
+        class="account"
+        (click)="switch(account.key)"
+      >
         <app-avatar
-          [text]="a.value.profile.name ?? a.value.profile.email"
-          [id]="a.value.profile.userId"
+          [text]="account.value.name ?? account.value.email"
+          [id]="account.value.id"
           [size]="25"
           [circle]="true"
           [fontSize]="14"
           [dynamic]="true"
-          [color]="a.value.avatarColor"
-          *ngIf="a.value.profile.email != null"
+          [color]="account.value.avatarColor"
+          *ngIf="account.value.email != null"
           aria-hidden="true"
         ></app-avatar>
         <div class="accountInfo">
           <span class="sr-only">{{ "switchAccount" | i18n }}:&nbsp;</span>
-          <span class="email">{{ a.value.profile.email }}</span>
-          <span class="server" *ngIf="a.value.serverUrl != 'bitwarden.com'">
-            <span class="sr-only"> / </span>
-            {{ a.value.serverUrl }}
-          </span>
-          <span class="status">
-            <span class="sr-only">&nbsp;(</span>
-            {{
-              (a.value.profile.authenticationStatus === authStatus.Unlocked ? "unlocked" : "locked")
+          <span class="email" aria-hidden="true">{{ account.value.email }}</span>
+          <span class="server" aria-hidden="true"
+            ><span class="sr-only"> / </span>{{ account.value.server }}</span
+          >
```

Comment:
The formatting here seems a bit confusing.
```suggestion
          <span class="server" aria-hidden="true">
            <span class="sr-only"> / </span>{{ account.value.server }}
          </span>
```